src/ckeditor/ckeditor.component.ts

```diff
@@ -291,7 +291,13 @@ export class CKEditorComponent implements AfterViewInit, OnDestroy, ControlValue
 		this.elementRef.nativeElement.appendChild( element );
 
 		if ( this.type === CKEditor4.EditorType.DIVAREA ) {
-			this.config = this.ensureDivareaPlugin( this.config || {} );
+			this.ensurePlugin( 'divarea' );
+		} else if ( this.config ) {
+			this.config.extraPlugins = this.removePlugin( this.config.extraPlugins, 'divarea' );
+		}
+
+		if ( this.type === CKEditor4.EditorType.INLINE ) {
+			this.ensurePlugin( 'floatingspace' );
 		}
```

Comment:
We are not adding `floatingspace` plugin by default in CKEditor 4, React, and Vue integrations, so I suppose there is no need to do exception here.

Reply:
I turned out that `divarea` plugin was also necessary in the second sample.